diff --git a/index.js b/index.js
index 5ae40cde5998f371490816ea1fa0b26df8ab6c5c..f79e41d8a88c05df419fa3223f1098f459586bc7 100644
--- a/index.js
+++ b/index.js
@@ -53,7 +53,7 @@ module.exports = (nextConfig = {}) => {
         customWorkerDir = 'worker',
         subdomainPrefix, // deprecated, use basePath in next.config.js instead
         ...workbox
-      } = pwa
+      } = nextConfig.pwa
 
       if (typeof nextConfig.webpack === 'function') {
         config = nextConfig.webpack(config, options)